lazycoder1412/EvaluateDivision.cpp

```diff
@@ -0,0 +1,34 @@
+class Solution {
+public:
+    map<string,vector<pair<string,double> > >graph;
+    map<string,bool> vis;
+    vector<double> ans;
+    double dfs(string start,string end){
+        vis[start]=true;
+        if(start==end)
+            return 1.0;
```

Comment:
if constants like 1.0 and -1.0 have a special meaning, a good practice is to declare them as constant variables with meaningful variable names within the class eg. `const float WIN = 1.0;`